### ISSUE TITLE
TeamPass 2FA QR Code won't show after providing activiation code

### ISSUE THREAD

**Issue by ajurna** [nilsteampassnet/TeamPass#2245]
### Steps to reproduce
1. Install Teampass
2. setup google 2fa and a user to use it.
3. email activation code.
4. login with activation code and login fails.

### Expected behaviour
QR Code is shown to enable 2FA

### Actual behaviour
cog spins

### Server configuration
**Operating system**:
ubuntu 18.04 LTS
**Web server:**
apache 2.4
**Database:**
mysql 5.7
**PHP version:**
7.2
**Teampass version:** 
2.1.27.12
**Teampass configuration file:**

**Updated from an older Teampass or fresh install:**
PLEASE attach to this issue the file `/includes/config/tp.config.php`.

### Client configuration
**Browser:**
chrome or firefox
**Operating system:**
win10
#### Web server error log
```
Insert your webserver log here
```
nothing relevant
#### Log from the web-browser developer console (CTRL + SHIFT + i)
response from identify.php
```
00", "error" : **"ga_temporary_code_correct"}]
```

**Comment by ajurna** [comment]
i'm afraid there really isn't an error. it's failing to make the correct string in the json reply.
![image](https://user-images.githubusercontent.com/883764/39617316-1f767fc6-4f77-11e8-9dc2-d4470c5e0b1a.png)
i tested this by running 2.1.27.11 and it's working fine there. it's probably an error on line 920 of identify.php

**Comment by nilsteampassnet** [comment]
Yes you are right.
The only change introduced here is a comment for code analyzer.

Current code is:
`echo '[{' +'"value" : "<img src=\"'.$new_2fa_qr.'\">", ' +'"user_admin":"', /** @scrutinizer ignore-type */ isset($_SESSION['user_admin']) ? +$antiXss->xss_clean($_SESSION['user_admin']) : "", '", ' +'"initial_url" : "'.@$_SESSION['initial_url'].'", "error" : "'.$logError.'"}]';`

Can you remove the ` /** @scrutinizer ignore-type */ ` and make a new try?

**Comment by nilsteampassnet** [comment]
OK I found the issue.

The fix will be committed in 2.1.27.13

Quick fix:
Open file `/sources/identify.php`
Replace
```
                    echo '[{' +'"value" : "<img src=\"'.$new_2fa_qr.'\">", ' +'"user_admin":"', /** @scrutinizer ignore-type */ isset($_SESSION['user_admin']) ? +$antiXss->xss_clean($_SESSION['user_admin']) : "", '", ' +'"initial_url" : "'.@$_SESSION['initial_url'].'", "error" : "'.$logError.'"}]';
```
by
```
                    echo '[{"value" : "<img src=\"'.$new_2fa_qr.'\">", "user_admin":"', /** @scrutinizer ignore-type */ isset($_SESSION['user_admin']) ? $antiXss->xss_clean($_SESSION['user_admin']) : "", '", "initial_url" : "'.@$_SESSION['initial_url'].'", "error" : "'.$logError.'"}]';
```

**Comment by nilsteampassnet** [comment]
Fixed.
Reopen on need.
